geneate_dynein_conformation_graphs/format_inital.py

```diff
@@ -0,0 +1,272 @@
+import csv
+import os
+import pandas as pd
+import matplotlib.pyplot as plt
+import pandas as pd
+import numpy as np
+import yaml
+
+rawdata_folder = 'Data'
+analysis_folder = 'Results'
+metadata_file = 'metadata.yaml'
+
+# Ensure output folder exists
+if not os.path.exists(metadata_file):
+    os.makedirs(analysis_folder)
+
+def transform_value_c(x):
+    return ((x - 1) % 9) + 1
+
+def transform_value_cc(x):    
+    match x:
+        case -7:
+            return 9
+        case -6:
+            return 8
+        case -5:
+            return 7
+        case -4:
+            return 6
+        case -3:
+            return 5
+        case -2:
+            return 4
+        case -1:
+            return 3
+        case 0:
+            return 2
+        case 1:
+            return 1
+        case 2:
+            return 9
+        case 3:
+            return 8
+        case 4:
+            return 7
+        case 5:
+            return 6
+        case 6:
+            return 5
+        case 7:
+            return 4
+        case 8:
+            return 3
+        case 9:
+            return 2
```

Comment:
This can be shortened to 
```suggestion
    return (-(x-1) % 9) +1
```